src/helpers/DevNote

```diff
@@ -0,0 +1,6 @@
+To protect a route, kindly import the permit middleware from the users file in the middlewares folder. 

```

Comment:
if you want to put a note, you can either put it in the `ReadMe.md` file or make this `DevNote` file a `DevNote.md` file.